src/style.css

```
@@ -1 +1,61 @@
 /* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding-left: 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  padding-top: 10px;
+}
+
+.nav__list {
+  box-sizing: border-box;
+  height: 60px;
```

```suggestion
  height: 60px;
```
Висоту 60 пікселів краще задати саме для тегу `<a>`, тоді і after буде відображатись для тестів коректно